include/common/debug.h

```diff
@@ -8,12 +8,14 @@
 
 #include <compiler.h>
 #include <stdint.h>
+#include <test.h>
```

Comment:
This is not needed here after you put `run_tests()` in the correct file

Reply:
done